src/main/java/org/gridsuite/ds/server/service/DynamicSimulationService.java

```diff
@@ -97,8 +101,12 @@ public Mono<DynamicSimulationStatus> getStatus(UUID resultUuid) {
 
     public Mono<Void> deleteResult(UUID resultUuid) {
         Objects.requireNonNull(resultUuid);
-        return Mono.fromRunnable(() -> resultRepository.deleteById(resultUuid))
-                .doOnError(throwable -> LOGGER.error(throwable.toString(), throwable)).then();
+        ResultEntity resultEntity = resultRepository.findById(resultUuid).orElseThrow();
+        // call time series client to delete timeseries and timeline
+        return Mono.zip(timeSeriesClient.deleteTimeSeriesGroup(resultEntity.getTimeSeriesId()).subscribeOn(Schedulers.boundedElastic()),
```

Comment:
Maybe you can subscribe only one time in the resulting mono?

Reply:
Done